R/qc_missed_cleavages.R

```diff
@@ -0,0 +1,58 @@
+#' Check number of missed cleavages
+#'
+#' Calculates the percentage of missed cleavages for each sample (by count).
+#'
+#' @param data A dataframe containing at least sample names, peptide or precursor identifiers and missed cleavage counts for each peptide or precursor.
+#' @param sample The column in the data dataframe containing the sample name.
+#' @param grouping The column in the data dataframe containing either precursor or peptide identifiers.
+#' @param missed_cleavages The column in the data dataframe containing the counts of missed cleavages per peptide or precursor.
+#' @param intensity Optional column containing the corresponding intensity values to each peptide or precursor.
```

Comment:
I would still add the code for computing the same statistic but based on intensity instead of count. This is what the intensity column is for.